src/chocolatey/infrastructure.app/domain/ChocolateyPackageInformation.cs

```diff
@@ -37,5 +37,6 @@ public ChocolateyPackageInformation(IPackageMetadata package)
         public bool IsPinned { get; set; }
         public string ExtraInformation { get; set; }
         public string DeploymentLocation { get; set; }
+        public string PackageInstalledFrom {  get; set; }

```

Comment:
Naming things is hard... Should this perhaps be `SourceInstalledFrom`? We can't necessarily use Uri as it may not be a Uri? Perhaps `PackageOriginationSource`?

Reply:
`SourceInstalledFrom` is clear what it contains, so I'd suggest that.